packages/core/content-manager/server/src/history/services/history.ts

```diff
@@ -97,8 +116,10 @@ const createHistoryService = ({ strapi }: { strapi: LoadedStrapi }) => {
         orderBy: [{ createdAt: 'desc' }],
       });
 
+      const localeDictionary = await this.getLocaleDictionary();
```

Comment:
It probably won't make a big diff but you could do this and the findPage above in a Promise.all